packages/nimble-components/src/table/components/header/styles.ts

```diff
@@ -1,6 +1,21 @@
 import { css } from '@microsoft/fast-element';
 import { display } from '@microsoft/fast-foundation';
+import {
+    applicationBackgroundColor,
+    controlHeight,
+    tableHeaderFont,
+    tableHeaderFontColor
+} from '../../../theme-provider/design-tokens';
 
 export const styles = css`
     ${display('flex')}
+
+    :host {
+        height: ${controlHeight};
+        align-items: center;
+        background: ${applicationBackgroundColor};
+        padding: 0px 8px;
```

Comment:
Is it worth changing `8px` to the calculated padding using `${standardPadding}`?

Reply:
Maybe? I'm never quite sure when to use a modified token vs set a hard-coded value. But, I think in this case it would be better to calculate because then a client could control the padding with a token if they need/want to.

Reply:
IMO, it'd be great to use a token if we have one (which it looks like we do with standardPadding)

@mollykreis , would we lose the ability for a client to override that with their own token if we went that route? I'm probably missing.

Reply:
@nate-ni, yes, using the standard padding token allows a client to override it. Eventually, we may want a token for "medium padding" because right now we only have tokens that default to 4px and 16px, but there are a number of places where we want to use the value of 8px.